navigation_control_node:
  ros__parameters:
    use_sim_time: false
    goal_tolerance_dist: 0.2
    goal_tolerance_angle: 0.1
    max_linear_velocity: 0.5
    max_angular_velocity: 1.0
    p_gain_linear: 0.8
    p_gain_angular: 1.5
